test/mk/compiler.mk

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: Apache-2.0 OR ISC OR MIT
+#
+# Compiler feature detection for mldsa-native
+# This file detects whether the compiler supports various architectural features
+# used by mldsa-native through compile-time tests with C code containing inline assembly.
+#
+# Feature detection can be overridden by setting the corresponding variable on the command line:
+#   make MK_COMPILER_SUPPORTS_SHA3=0    # Disable SHA3 detection
+#   make MK_COMPILER_SUPPORTS_AVX2=0    # Disable AVX2 detection
+#   make MK_COMPILER_SUPPORTS_BMI2=0    # Disable BMI2 detection
+#   make MK_COMPILER_SUPPORTS_SSE2=0    # Disable SSE2 detection
+
+ifndef _COMPILER_MK
+_COMPILER_MK :=
+
+# Override ARCH for cross-compilation based on CROSS_PREFIX
+ifeq ($(CROSS_PREFIX),)
+# Normalize architecture names
+ARCH := $(shell uname -m)
```

Comment:
When I diff this file to mlkem-native's compiler.mk I see minor whitespace and order differences. That makes syncing these files in the future unnecessarily hard. Could you align these two please?